src/engine/source/kvdb2/include/kvdb2/iKVDBHandlerManager.hpp

```diff
@@ -1,18 +1,19 @@
 #ifndef _IKVDBHANDLERMANAGER_H
 #define _IKVDBHANDLERMANAGER_H
 
+#include <kvdb2/iKVDBHandler.hpp>
 #include <memory>
 #include <string>
-#include <kvdb2/iKVDBHandler.hpp>
 
 namespace kvdbManager
 {
 
 class IKVDBHandlerManager
 {
 public:
-    virtual std::shared_ptr<IKVDBHandler> getKVDBHandler(const std::string& dbName, const std::string& scopeName) = 0;
+    virtual KVDBHandler getKVDBHandler(const std::string& dbName, const std::string& scopeName) = 0;
```

Comment:
I think it would be better to using here the whole data type instead of its alias